README.md

```diff
@@ -1 +1,68 @@
-# awesomeBooksES6
\ No newline at end of file
+# "Awesome books: with ES6"
+
+> In this project, I built a basic website that allows users to add/remove books from a list. I achieved that by using JavaScript objects and arrays, I also dynamically modified the DOM and added basic events.This is a basic SPA (Single Page Application) which is created by JavaScript classes and manipulated by DOM elements and JavaScript events. Simple USER INTERFACE that allows user to ADD and REMOVE books.
+
+## Screenshot
+
+![Screenshot (773)](https://user-images.githubusercontent.com/19844175/156057583-f7b24449-da50-499a-ae8a-8e689f3356a2.png)
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- Linters
+
+## Live Demo
+
+[https://mutalenic.github.io/awesomeBooksES6/]
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- GitHub/VSCode/Linters
+
+### Setup
+
+- gh repo clone [https://github.com/Mutalenic/awesomeBooksES6.git]
+
+## Authors
+
+ 👤 **Nicholas Mutale** https://github.com/Mutalenic
+ 
+ - GitHub: [@GitHub/Mutalenic](https://github.com/Mutalenic)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page [https://github.com/Mutalenic/awesomeBooksES6/issues].
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+👤 **Said Rasinlic**
+
+- GitHub: [@GitHub/SaidRasinlic](https://github.com/SaidRasinlic)
+- Twitter: [@Twitter/SaidRasinlic](https://twitter.com/SaidRasinlic)
+- LinkedIn: [LinkedIn/SaidRasinlic](https://www.linkedin.com/in/saidrasinlic)
+
+👤 **Mehdi Rahal**
+
+ - GitHub: [@GitHub/Mehdi-Rh](https://github.com/Mehdi-Rh)
+
+
+- Microverse 
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- Please consider having a completely professional README file and fix the broken MIT link. Keep in mind that this can leave a bad impression on the viewers of your project 😄